pkg/controller/kieapp/constants/constants.go

```diff
@@ -177,6 +178,10 @@ const (
 	ConsoleAuthoringMemRequests = "3Gi"
 	ConsoleProdCPURequests      = "500m"
 	ConsoleProdMemRequests      = "1500Mi"
+	DashbuilderCPULimit         = "1"
+	DashbuilderCPURequests      = "750m"
+	DashbuilderMemLimit         = "2Gi"
+	DashbuilderMemRequests      = "1536m"
```

Comment:
not sure but I don't think mem request works with `m` can be changed to `1500Mi` to follow the pattern?

Reply:
good point, it should be Mi,